packages/kernel/src/worker-runtime.ts

```diff
@@ -396,6 +398,9 @@ AppSession._on_scriptrunner_event = wrap_app_session_on_scriptrunner_event(AppSe
 
     const canonicalEntrypoint = resolveAppPath(appId, entrypoint);
 
+    postProgressMessage("Importing Language Server");
+    await importLanguageServerLibraries(pyodide, micropip);
```

Comment:
It would be great if the language-server feature is disabled by default and we can opt-in it by an option because the users of Stlite as a library (`@stlite/browser` and `@stlite/desktop`) don't need to import these packages.

Reply:
That works, I can add a parameter in the options. Let me know if you prefer something else.

Reply:
Added new option `languageServer` in the `StliteKernelOptions` that will control if the libraries will be loaded or not. The `languageServer` name made sense to me because later it can be extended to support passing options to configure the language server or boolean to turn on/off completely. 
I don't have a strong opinion about the name, so not a problem to use a different name here.

Reply:
Thank you, I agree with the `languageServer` name.